src/components/DataProvider/index.js

```diff
@@ -602,6 +604,7 @@ DataProvider.propTypes = {
   // xSubDomain => newXSubDomain
   // function to allow limitation of the value of xSubDomain
   limitXSubDomain: PropTypes.func,
+  onFetchData: PropTypes.func,
```

Comment:
What does this signature look like? What does it do? Why would I use it? (please jsdoc this)